src/style.css

```
@@ -1,3 +1,90 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: gray;
+  text-transform: uppercase;
+
+  --blue: rgba(0, 172, 220, 1);
+  --white: rgba(255, 255, 255, 1);
 }
+
+.header {
+  margin: 0 auto;
+  background-color: var(--white);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__image {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: auto 10px;
```

it's dangerous solution because there can be margin-collapse, please read about it and try to use margin20px from one side, example: 
```suggestion
 .nav__item:not(:last-child) {
   margin-right: 20px;
 }
```